src/Admin/AbstractAdmin.php

```diff
@@ -3154,6 +3154,8 @@ public function getDashboardActions()
     }
 
     /**
+     * NEXT_MAJOR: remove this method.
```

Comment:
You need to add a deprecation in the method.
You can check others `@trigger_error` message to find what to write.